internal/core/command/router.go

```diff
@@ -37,7 +38,7 @@ func LoadRestRoutes(dic *di.Container) *mux.Router {
 
 	// Configuration
 	r.HandleFunc(clients.ApiConfigRoute, func(w http.ResponseWriter, r *http.Request) {
-		configHandler(w, bootstrapContainer.LoggingClientFrom(dic.Get))
+		configHandler(w, bootstrapContainer.LoggingClientFrom(dic.Get), *container.ConfigurationFrom(dic.Get))
```

Comment:
Commented once; may apply elsewhere.

Don't copy the resulting configuration struct; pass a reference (and have configHandler continue to pass a reference down the call chain as needed).

Reply:
Removing the deference syntax throws an error about the underlying struct not adhering to the interface.